ocaml/xapi/xapi_host.ml

```diff
@@ -792,18 +792,12 @@ let management_reconfigure ~__context ~pif =
   if Db.Pool.get_ha_enabled ~__context ~self:pool then
     raise (Api_errors.Server_error(Api_errors.ha_is_enabled, []));
 
-  (* Plugging a bond slave is not allowed *)
-  if Db.PIF.get_bond_slave_of ~__context ~self:pif <> Ref.null then
```

Comment:
I think a flag is needed for checking `required_address_type <> primary_address_type` in case of `xapi_pool` assert. This is not needed for `xapi_host`.